ART_NOTES.md

```diff
@@ -0,0 +1,112 @@
+
```

Comment:
Could we have a TLDR for say "Steps for Sam who wants to demo Boss Room" and "Steps for Sam when he wants to edit the scene"?

Reply:
In other words, users like me could be pretty inexperienced with lighting and baking (and wouldn't want to start learning about this in a Netcode project...). So having concise instructions at the tope to get you where you want to go could be best.

Reply:
Ahaha can do!

Reply:
Can you and @s-omeilia-unity review this together?
I tried to create an iOS build for my low end iPhone device, but I was confused as to what I needed to do to make sure it was running smoothly.

Reply:
Will read through this today!